wapitiCore/net/crawler.py

```diff
@@ -231,7 +232,11 @@ async def async_get(
         @type stream: bool
         @rtype: Response
         """
-        request = self._client.build_request("GET", resource.url, headers=headers, timeout=self.timeout)
+        if timeout is None:
+            request = self._client.build_request("GET", resource.url, headers=headers, timeout=self.timeout)
+        else:
+            timeout = httpx.Timeout(timeout)
+            request = self._client.build_request("GET", resource.url, headers=headers, timeout=timeout)
```

Comment:
a more compact way is 
```suggestion
        timeout = self.timeout if timeout is none else httpx.Timeout(timeout)
        request = self._client.build_request("GET", resource.url, headers=headers, timeout=timeout)

```